examples/pup/motor/motor_until_stalled_center.py

```diff
@@ -16,7 +16,7 @@
 # We have just moved to the rightmost endstop. So, we can reset
 # this angle to be half the distance between the two endpoints.
 # That way, the middle corresponds to 0 degrees.
-example_motor.reset_angle((right_end - left_end) / 2)
+example_motor.reset_angle((right_end - left_end) // 2)
```

Comment:
Actually, the correct fix is to modify `reset_angle()` so that it allows `float`.